### ISSUE TITLE
[Query] Is this repo actively participating in SWOC

### ISSUE THREAD

**Issue by abhirajadhikary06** [sanchitc05/Trip-Tailor#1]
If this repo is participating in SWOC'25 I would like to add a great number of contributions to this repo.

**Comment by sanchitc05** [comment]
@abhirajadhikary06 Yes this repo is participating in SWOC. Stay prepared to make worthy contributions to it 😀

**Comment by abhirajadhikary06** [comment]
Have you planned what contents to add. Which framework to use. Procedure of deployment ?
